### PR TITLE
Fix escapedUnicodeString for css

### PR DESCRIPTION
**Pull request by stevenbfine**
Non ascii unicode characters need to be escaped differently in HTML and CSS.

### CODE REVIEW

File: HTMLSerializer.js [hunk 1]
```diff
@@ -397,15 +397,19 @@ var HTMLSerializer = class {
    * characters escaped.
    *
    * @param {string} str The string that should have its characters escaped.
+   * @param {boolean} isHTML true if the string being escaped is in html. false
+   *     if css.
    * @return {string} The correctly escaped string.
    */
-  escapedUnicodeString(str) {
+  escapedUnicodeString(str, isHTML) {
```
Comment (chrishtr) [inline HTMLSerializer.js]: Use an "enum" instead:

UnicodeEscapeInputType = {
    HTML : 0,
    CSS : 1,
}

Comment (stevenbfine) [inline HTMLSerializer.js]: Done

File: HTMLSerializer.js [hunk 2]
```diff
@@ -59,6 +59,16 @@ var HTMLSerializer = class {
     };
 
     /**
+     * @public {Object<string, number>} An enum representing different types
+     *     text.
```
Comment (chrishtr) [inline HTMLSerializer.js:63]: "of text"

Comment (stevenbfine) [inline HTMLSerializer.js:63]: Done

**Comment by progers** [comment]
Nice, LGTM

**Pull request merged**